index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header>
+            <h2 class="logo">My Logo</h2>
+            <img class="hamburger" src="images/hamburger-menu.png" alt="hamburger menu" width=60>
+        </header>
+        
+        <section class="headline">
+            <h2>I am Jerry <br> Glad to see you!</h2>
+            <p class="about-my-self">I’m a software developer! I can help you build a product ,
+                 feature or website Look through some of my work and 
+                 experience! If you like what you see and have a project
+                  you need coded, don’t hestiate to contact me.</p>
+        
+        
+            <a href="#contacts">LET'S CONNECT</a>
+
+            <div class="contacts">
+                <a href="">
+                    <img src="images/twitter.svg" alt="twitter" width=20>
+                </a>
+                <a href="">
+                    <img src="images/linkedin-i.png" alt="linkedin" width=8>
+                    <img src="images/linkedin-n.png" alt="linkedin" width=15>                  
+                </a>
+                <a href="">
+                    <img src="images/m.svg" alt="microverse" width=25>
+                </a>
+                <a href="">
+                    <img src="images/github.svg" alt="github" width=20>
+                </a>
+                <a href="">
+                    <img src="images/two-finger.svg" alt="two-finger" width=20>
+                </a>
```

Comment:
Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).